packages/@aws-cdk/aws-events-targets/lib/ecs-task.ts

```diff
@@ -117,33 +126,8 @@ export class EcsTask implements events.IRuleTarget {
    * Allows using tasks as target of EventBridge events
    */
   public bind(_rule: events.IRule, _id?: string): events.RuleTargetConfig {
-    const policyStatements = [new iam.PolicyStatement({
```

Comment:
Please revert this. I get that you don't want this for your specific use case right now, but we have to have consistent behavior across the whole of CDK otherwise people will constantly have to be digging in documentation.

If you want the specific role you're passing to not have statements added to them, pass `role.withoutPolicyUpdates()` instead.

Reply:
Looks like you thumbs-upped my comment, but didn't actually make the change.

Is the code now in a situation where the policy statements are added to the role users pass in (if they do)?

Reply:
Oops! No, it isn't. I'm agree to your advice and this code should be fix.
But I haven't been able to do anything because I've been so busy lately, unlike a month ago. so sorry for late to fix it.

Reply:
so I fixed it. I'm happy if you check it. sorry to late for the fix.